website/content/v1.7/kubernetes-guides/configuration/storage.md

```diff
@@ -96,6 +96,16 @@ First create a config patch file named `mayastor-patch.yaml` with the following
   path: /machine/nodeLabels
   value:
     openebs.io/engine: mayastor
+- op: add
+  path: /machine/kubelet/extraMounts
+  value:
+    - destination: /var/local
+      type: bind
+      source: /var/lib/local
```

Comment:
While mayastore recommends `/var/lib/local` i think `/var/lib/openebs` might be better.

Reply:
it should match I guess something in the installed helm chart?

Reply:
I am suggesting for the source. Lets leave destination as it is.